cmd/ci-operator/main.go

```diff
@@ -186,6 +183,9 @@ func main() {
 		if err := flag.CommandLine.Set("v", "10"); err != nil {
 			logrus.WithError(err).Error("Failed to set flag -v=10")
 		}
+		logrus.SetLevel(logrus.TraceLevel)
```

Comment:
?

Reply:
This was part of my commit to add logging to the artifacts thing. The default log library we use has no concept of levels, and we use logrus anyway, so I just added logging to it in Trace level, and setting ci-operator verbose mode opens that up.

Reply:
I think my larger point was that almost everywhere we run logrus with Debug level by default and reserve trace level logs for things that you need to opt in to get them, I think it is valuable to be consistent with that

Reply:
Well this *is* opt-in, it's under verbose so I guess this is fine?

Reply:
Yeah you need to `--verbose` the `ci-operator` binary to get this, which you'd only be doing in dev.